dataplicity/tags.py

```diff
@@ -1,35 +1,41 @@
 from __future__ import unicode_literals
 
+import os
 import subprocess
 import re
 import logging
 
 log = logging.getLogger("agent")
 
 
-TAG_SCRIPT = "/home/dataplicity/dataplicity_tags"
-
-
-class TagException(Exception):
+class TagError(Exception):
     """Custom exception raised when get_tag_list has an exception"""
 
-    pass
-
 
 def get_tag_list():
     """Run the dataplicity.tags script, get output as a list of tags"""
+
```

Comment:
Use the HOME env var to get the script location. Add a bit of logging.

Reply:
hmm wouldn't it be better to allow the user to specify the directory instead of forcing the user to overwrite $HOME ? it may be benine, but I think that $HOME is used on the system, so that's why I'd be cautious about it. What I mean is - what if the user installs dataplicity to the default location, but he simply prefers to have the tags in some other directory? Is that even possible or impossible?

Reply:
Quite possibly. We don't have an easy way to configure agent at the moment. I think one user must have a customized configuration.